_posts/2017-05-10-april.md

```diff
@@ -0,0 +1,86 @@
+
+```javascript
+import {Program} from "witheve";
+
+let program = new Program("my program");
+let htmlWatcher = program.attach("html");
+
+let someElement = document.querySelector("#eve-wrapper");
+htmlWatcher.addExternalRoot("my-root", someElement);
+``` 
+
+From within Eve, you can search for that element and add children to it, as you would any other record:
+
+```eve
+search
+  root = [#my-root]
+
+bind
+  root.children += [#ui/text text: "Hello!"]
+```
+
+Eve behaves as expected, updating the embedded element as records change. For new users, this is a nice way to introduce Eve to an existing project; instead of porting an application from scratch, you can dip your toes in the water and build a new component in Eve, then inject it into your project wherever you want. As you get more comfortable with Eve, you can use it in more of you application.
+
+#### Eve Syntax Returns
+
+When we initially released the v0.3 preview, it notably didn't include a parser for the Eve syntax. We had planned on eventually adding the syntax back, but it wasn't our main priority because the Javascript DSL served as an adequate replacement for the time being. However, because we received a number of requests to add it back in, we decided getting the syntax parser hooked up was a higher priority. That work is generally completed, although it's not merged into the main branch yet. The next preview release will include the parser, and you'll be able to write Eve code in a JavaScript program, or parse a stand-alone *.eve file. 
+
+I'd like to say thinks to everyone who gave their input regarding the syntax. When we released it with v0.2, we didn't know how it would be received, so we were happy to see that people liked it enough to request it return with v0.3. As always, we're developing Eve in the open and with our community, so your feedback has a big impact on how we prioritize work.
+
+### Web Presence
+
+Before we can officially release v0.3, a lot of work needs to be done on the usability and documentation side of things. To this end, we revamped our website to make it more accessible to people unfamiliar but curious about Eve. A big part of this effort was actually coming up with a one-sentence pitch, describing Eve in a nutshell.
+
+#### A new pitch for Eve
+
+Here's our new one sentence pitch for Eve:
+
+> Eve is a modern relational language for writing data-driven programs without the boilerplate.
+
+The intended audience for this pitch is generally the "HackerNews" crowd -- developers who are curious about languages and likely to something new to the scene. Eve has many aspirations (being a language for non-programmers), but this pitch is directed more toward explaining Eve in its current form to programmers. We wanted to pack as much information into this one sentence as possible, so you can get a general idea of the concepts important to Eve after reading it. This is actually easier said than done, so I'll break down how we went about this.
```

Comment:
"Eve has many aspirations (being a language for non-programmers)" - i think "such as being a language" reads better, because we have many aspirations, not just one